src/Compilers/CSharp/Portable/Symbols/TypeSymbolWithAnnotations.cs

```diff
@@ -414,7 +402,7 @@ internal static TypeSymbolWithAnnotations Create(bool isNullableEnabled, TypeSym
 
         internal static TypeSymbolWithAnnotations Create(TypeSymbol typeSymbol, NullableAnnotation nullableAnnotation = NullableAnnotation.Unknown, ImmutableArray<CustomModifier> customModifiers = default)
         {
-            if (typeSymbol is null)
+            if (typeSymbol is null && nullableAnnotation == 0)
```

Comment:
>0 [](start = 60, length = 1)

I don't think someone should pass an "unset" value for `nullableAnnotation` here, so should probably be `Unkonwn`.

Reply:
I'm specifically testing for the default value of the enum to determine if we can use the default value of TypeSymbolWithAnnotations.  Checking for 0 is therefore the intent.

---
In reply to: [260036008](https://github.com/dotnet/roslyn/pull/33648#discussion_r260036008) [](ancestors = 260036008)

Reply:
@gafter you could use `default` as the comparison, so it's more clear what you're testing for.

---
In reply to: [260058148](https://github.com/dotnet/roslyn/pull/33648#discussion_r260058148) [](ancestors = 260058148,260036008)